frontend/src/components/Header/index.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { Button } from './../shared/button/index.js';
+import { Heading, HEADING_OPTIONS } from '../shared/heading';
+import { useEffect, useState } from 'react';
+import { API } from '../../modules/apis';
+
+// import { Navbar } from './../../components/Navbar/index.jsx';
+
+const Header = () => {
+  const [data, setData] = useState();
+  useEffect(() => {
+    API('/header').then(({ data }) => {
+      setData(data);
+    });
+  }, []);
+  if (data) {
+    return (
+      <header className=" w-full  bg-c400  border-c100 border-b-4 pb-32">
+        <div className="container flex flex-row pt-64 relative">
+          <div className="  mt-16  ">
+            <div className="  ">
+              <span className=" text-c100 font-body text-xxlg font-medium leading-1 ">
+                I am
```

Comment:
You missed a spot there !

Reply:
you meant empty class name?

Reply:
'I am' is static it shouldn't be

Reply:
I ask nagawy and he say let him

Reply:
@AbdoMgdy  what do you say?

Reply:
if nag3away approve then i'm ok with it

Reply:
then accept your request changes and he will decide
to approve or not as you depend on him